config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  resources :users
+
+  root "users#index"
+
+  resources :users do
+    resources :posts
+  end
   # Defines the root path route ("/")
   # root "articles#index"
 end
```

Comment:
- [ ] As required for the project, kindly make sure `users` and `posts` routes added to your `config/routes.rb` maintained a RESTful URL corresponding to the listed **[wireframes from your project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project.md)**, i.e you do not expose unnnecessary endpoints only the 4 required URLs should be there(may be the exra `route` can be for the `root` one as you did). Once you have edited you routes configuration, you can run `rails routes | grep user` in your terminal and it should yield URLs that correspond to [these wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project.md):
You can refer this **[article](https://guides.rubyonrails.org/routing.html#configuring-the-rails-router)** to get an example on how to do that

Reply:
I have fixed this issue, thank you @tresorsawasawa! 🙌🏽